### PR TITLE
Branch enhanced UI

### LINKED ISSUE #40: As a Computer Science Job Seeker, I can easily visualise the job contacts (UI)
Enchances the UI by replacing some design elements

### PR DESCRIPTION
**Pull request by Dexter-Wong**
Enhance readabality of the app

Fix #40

### CODE REVIEW

File: src/main/resources/view/DarkTheme.css [hunk 1]
```diff
@@ -149,8 +149,8 @@
 .result-display {
     -fx-background-color: transparent;
     -fx-font-family: "Segoe UI Light";
-    -fx-font-size: 13pt;
-    -fx-text-fill: white;
+    -fx-font-size: 20pt;
+    -fx-text-fill: rgb(81, 255, 0);
```
Comment (dabzpengu) [inline src/main/resources/view/DarkTheme.css:153]: nice colour!

File: src/main/resources/view/DarkTheme.css
Comment (dabzpengu) [inline src/main/resources/view/DarkTheme.css:1]: nice OOP!

File: src/main/resources/view/CustomTheme.css
Comment (ZHANGTIANYAO1) [inline src/main/resources/view/CustomTheme.css:1]: Nice color

File: src/main/resources/view/DarkTheme.css
Comment (ZHANGTIANYAO1) [inline src/main/resources/view/DarkTheme.css:1]: Nice UI

File: src/main/java/seedu/address/logic/commands/AddCommand.java
Comment (ZHANGTIANYAO1) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:1]: Nice change

**Review by ZHANGTIANYAO1 — COMMENTED** [review]
LGTM

**Pull request merged**